samples/snippets/csharp/language-reference/keywords/interpolated-strings5.cs

```diff
@@ -0,0 +1,20 @@
+// <Snippet1>
+using System;
+
+public class Example
+{
+    public static void Main()
+    {
+        var name = "Horace";
+        var age = 34;
+        var s1 = $@"He asked, ""Is your name {name}?"", but didn't wait for a reply.
+ {name} is {age:D3}
+        year{(age == 1 ? "" : "s")}
+        old.";
```

Comment:
Can these three lines be on a single line? I think that it makes this code very difficult to read.